apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -112,7 +112,7 @@ const defaultValues: IForm = {
   description: '',
   identifier: '',
   tags: [],
-  critical: false,
```

Comment:
changed default value to true so that switch button for critical option is on and workflow is created non-critical